File: JAVA/PARCIAL II/EVA-POO-TrujilloDavid/EVA-POO-TrujilloDavid/Entrenador.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EVA_POO_TrujilloDavid
{
	internal class Entrenador:Persona , IEntrenador
	{
		private string federacion;

		public Entrenador(int id, string nombre, int edad, string federacion): base(id, nombre, edad)
		{
			this.federacion = federacion;
		}
		public string Federacion
		{
			get { return federacion; }
			set { federacion = value; }
		}
		//private string Inicio()
		//{
		//	return "El entrenador " + base.Nombre;
		//}
		public override string Concentrarse()
		{
			return "El entrenador " + base.Nombre + " se esta concentrando...";
		}

		public override string Viajar()
		{
			return "El entrenador " + base.Nombre + " esta viajando";
		}

		public string DirigirPartido()
		{
			return "El entrenador "+base.Nombre+" esta dirigiendo el partido";
		}

		public string DirigirEntrenamiento()
		{
			return "El entrenador " + base.Nombre + " esta dirigiendo el entrenamiento";  
		}

		public override string ToString()
		{
			return base.ToString() + " Fedaracion: " + federacion;
		}
	}
}

File: JAVA/PARCIAL II/EVA07-POO-TrujilloDavid/src/paquete1/Entrenador.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package paquete1;

/**
 *
 * @author Asus
 */
public class Entrenador extends Persona{
    protected String federacion;

    public Entrenador(int id, String nombre, int edad, String federacion) {
        super(id, nombre, edad);
        this.federacion = federacion;
    }
    
    public String DigirPartido(){
        return "El entenador "+super.nombre+" esta dirigiendo un partido";
    }
    public String DirigirEntrenamiento(){
        return "El entenador "+super.nombre+" esta dirigiendo un entrenamiento";
    }
    @Override
    public String Concentrarse() {
        return "El entrenador "+super.nombre+" se esta concentrando";
    }

    @Override
    public String Viajar() {
        return "El entrenador "+super.nombre+" esta viajando";
    }
    
    @Override
    public String toString(){
        return super.toString()+" Federacion: "+federacion;
    }
}
